rotkehlchen/db/ethtx.py

```diff
@@ -259,19 +264,26 @@ def get_transaction_hashes_no_receipt(
         return hashes
 
     def get_transaction_hashes_not_decoded(self, limit: Optional[int]) -> List[EVMTxHash]:
+        """
+        Get the transactions hashes for the transactions that have not been decoded. If the limit
+        argument is provided then it is used in the SQL query with the default order.
+        """
         cursor = self.db.conn.cursor()
-        querystr = (
-            'SELECT A.tx_hash from ethtx_receipts AS A LEFT OUTER JOIN evm_tx_mappings AS B '
-            'ON A.tx_hash=B.tx_hash WHERE B.tx_hash is NULL'
-        )
-        bindings = ()
+        querystr = 'SELECT A.tx_hash from ' + TRANSACTIONS_MISSING_DECODING_QUERY
```

Comment:
Since you are doing this put the cursor also under the context manager

Reply:
I thought that and forgot. Will do